### DISCUSSION TITLE
How to contribute a translation

### DISCUSSION THREAD

**Discussion by ElMorito** [CorentinTh/it-tools#895] [Q&A]
Hi there 🙋‍♂️

I would like to contribute a translation for this repository. I am a native speaker of German and I would be happy to help make the tools more accessible to people who speak my language.

I would like to know how I can submit my translations so that they can be included in the repository.

Thank you for your time and consideration.

**Comment by cwm-peace** [comment]
Hallo, du kannst das Lagerhaus von GitHub herunterziehen, einen neuen Zweig von der Hauptlinie aus starten, deine Entwicklung abschließen und deine Änderungen einreichen, um dann einen Pull-Request zu stellen. Nach der Überprüfung kannst du die Änderungen zusammenführen.

**Comment by steffenrapp** [comment]
@ElMorito Feel free to review my PR #1038 😃 I've translated everything to German that can be translated right now.
